### ISSUE TITLE
GC Page Template Usability: 'Date modified' field

### ISSUE THREAD

**Issue by neoinsight** [wet-boew/GCWeb#1061]
Attention @dk-tbs, @pcwsmith

In recent usability testing, participants were asked to find out when a document posted online had last been updated. Despite a link to 'View change log'  and a 'Last updated on' date higher on the page, several immediately scrolled down to check the 'Date modified: <date>" field and cited that date. Uh oh! 

In response, we moved the document date field closer to the 'Date modified' field at the bottom of the page to make it easier to notice and we did find that helped, as seen in the image below. 

![image](https://cloud.githubusercontent.com/assets/1855584/12521679/0903807e-c11a-11e5-834d-2dbf7ec1e89d.png)

However, for clarity around what has been updated,  and for plain language purposes, it would be wise to change 'Date modified' to 'Page updated on' or 'Page modified on' on all Canada.ca templates.

**Comment by pcwsmith** [comment]
Added to the to-do list, thx

**Comment by hsrudnicki** [comment]
Based on the above, this can be closed.
[In case if you think otherwise, please update the status of this again. Thank you]

**Comment by shawnthompson** [comment]
This change was never made. @pcwsmith Are you still planning on making this change?

**Comment by pcwsmith** [comment]
(sigh) haven't documented it in the specifications yet. but we're gonna, for reals!

**Comment by DK-TBS** [comment]
@shawnthompson - thanks for reopening this one.  We're planning to update this in the next version of the spec, so it won't be long now.  Might as well go ahead and make the change, if it's not too much work.  Maybe Mr Pepin can come up with a French equivalent for "page updated on" so you can tweak the demo templates?  Hopefully you only have to make the change in one place  :)

**Comment by davidpepin** [comment]
A French wording that would work would be "Page mise à jour le". 

But I am not sure this is a good idea. A page may have been “modified” (to fix a typo, for example), but not actually “updated”. In French, at least, “mise à jour” implies that the information was complete and current as of that date (and it may not be the case). As a user, if I see "Page updated on", I understand that this page was reviewed and updated in full on that date, and that it is fully accurate as of that date. Whereas the "Date modified" simply indicates that "something" has changed on the page, not that it was fully reviewed. 

Since the "Date modified" is updated automatically each time a page is republished, the wording "Page updated on" could be misleading.

**Comment by neoinsight** [comment]
Thanks @davidpepin for your input about 'updating'. You're right, that might be more general than intended. 
The issue here is that 'Date modified' is also overly general, and participants interpreted it to mean the document had been updated. That's why I was suggesting that it should be refined to 'Page' rather than the more general 'Date' - to be clearer about WHAT changed - that it was the page, not the document.  As to whether it says 'updated' versus 'modified', I was looking for a simpler more familiar term than 'modified'. 

So from what you are saying 'Page modified on' would be your preference.

**Comment by davidpepin** [comment]
Yes, "Page modified on" seems more accurate. But I'm not sure if it's really better than the well-known "Date modified" that we currently have.

**Comment by DK-TBS** [comment]
The thing that makes it better is the fact that it's a little more specific:  it's not the publication mentioned on the page that was updated, it's the page itself that was updated.  The "publication last updated on" date would be elsewhere on the page, closer to the publication itself...

Are we happy to run with "Page modified on" as a label?  It works well enough for me.

**Comment by davidpepin** [comment]
Works for me: 
Page modified on: 
Page modifiée le :

**Comment by neoinsight** [comment]
Works for me!

**Comment by pjackson28** [comment]
The challenge is with "Page modified on:" you would need to update the date anytime there are changes, such as a new template, any change to the HTML markup (including invisible markup changes, etc. The original intent was to indicate the currency of the content not of the web page itself so readers know when something substantive has changed in the content (so as to know when to read again). I'm not sure what use it would be to citizens to indicate that something invisible (and irrelevant from a reader standpoint) has been updated on the page.

**Comment by pjackson28** [comment]
If you are looking to be more specific, maybe something like "Content modified on:" or something along those lines would be better. The goal should be to target just the content and not the container itself.

**Comment by DK-TBS** [comment]
Oh yeah, I guarantee nobody is interested in knowing whether the page code was updated, and we're not interested in telling them about it either.  I'm guessing that when a user sees "page modified on", they'll assume it's the page content and not something under the hood.

"Content modified on" may be more specific, but my concern there is that most people probably don't use the word "content" the way we do.  A little jargonish.

I suppose we could go with something like "Page text modified on".  Everybody happy with that?

**Comment by pjackson28** [comment]
My concern with "Page text" is that web page content is not always text-based. It could also involve images, multimedia and other non-text content which when changed should affect the displayed date. I don't know what would be good wording but if we are going to be specific then it is important that we are also precise as this date could have significance beyond just identifying the currency of the content to the reader (e.g., could have policy implications).

**Comment by DK-TBS** [comment]
Okay then - how about "Contents of this page last modified on"?  A little wordy maybe, but we're kind of running out of options, I think  :)

**Comment by pjackson28** [comment]
I agree that it seems a bit wordy but at least it is precise. Perhaps something like "Page contents last modified:" of "Page contents modified:" could work better. Don't know how usable those are as options or if there any better options that are accurate, short and easy to understand.

**Comment by davidpepin** [comment]
A few things to consider: 
- By default, that page gets changed every time there is a modification on the page and the page gets republished. Not matter what was changed. It is, literally, the date anything on the page was last modified. "Date modified" conveys that perfectly well. 
- At Health Canada's request, we are adding the option of manually modifying that date. I believe this was a mistake. This date at the bottom has no legal implications or anything: it just says that something (anything) was changed on the page. If the date of the last full update is needed, it should be in the content of the page. 
- Any change to this will have to go through the proper channels (actual config change request). 

To be honest, I feel like we currently have WAY bigger fishes to fry with this system than changing a wording that conveys exactly what it is (the date something was modified). 

Also, "Page contents last modified" would be "Le contenu de cette page a été modifé le ". That's too long. And not accurate, because that's not what this date is for.

**Comment by davidpepin** [comment]
It's actually worse: "Le contenu de cette page a été modifé pour la dernière fois le".

**Comment by pjackson28** [comment]
@davidpepin Then the behavior of that field is out of sync with policy requirements as since 2000 (all the way back to CLF 1.0) it has been about when the page content was last substantially updated or modified.

As for saying the date at the bottom having no legal implications, I don't think that is an accurate statement as the currency of the content has meant things to certain policy instruments in the past (e.g., Web Standards) and the Date Modified was viewed as being a currency indicator.

**Comment by davidpepin** [comment]
The fact that users will be able to manually modify the "Date modified" field may cover the policy requirements you are mentioning, then. 

Still, when discussing these, the French equivalent has to be considered. Even in a shortened version ("Contenu modifié pour la dernière fois le", or even "Contenu modifié en date du", which would be translations of "Content last modified on"), it's too long (think mobile view). 

"Last modified:" and "Date de modification :" are totally fine as far as I am concerned. They are not "wrong". Of course I'm in no position to decide. I just feel like there are so many configuration needs right now, and so little resources, that this debate should be pushed to a later date. 

In the end, any request to change that wording will have to come from the right channels, and I believe that the Content & IA team on your side would need to look at it (maybe also the team responsible for the Content style guide? )

**Comment by davidpepin** [comment]
ok, the Standard on Web Usability does say: 
 **date modified (date de modification)

    The most recent date on which the Web page content was formally issued, substantially changed or reviewed.**

I stand corrected. 

I guess that to change the wording that mandatory element, the actual Standard would have to be modified?

**Comment by krisaston** [comment]
To add another policy aspect, the Standard on Metadata also has Date Modified as a mandatory element. I don't know what the status of the Std on Metadata is currently, but it's probably worth checking with CIOB IM folks.

**Comment by DK-TBS** [comment]
Yep, we all definitely have bigger fish to fry.  This was supposed to be a quick and easy thing, to help address the confusion that Lisa was seeing in some of the user testing.  @neoinsight @pcwsmith, feel free to keep the conversation going if you think it's important enough, but I'm happy to drop it.

**Comment by neoinsight** [comment]
I withdraw the recommendation. Yes, some participants were confused, but it's not a top task, it's not urgent compared to many of the other critical issues I've posted and I didn't realize there were so many other complications.  I will close the issue later today.
